src/components/BankDetails/RecipientDetails/RecipientDetailsForm/RecipientDetailsForm.tsx

```diff
@@ -245,9 +245,16 @@ export default function RecipientDetailsForm({
 
                   validate: f.validationAsync
                     ? async (v: string) => {
-                        const { params, url } = f.validationAsync!;
-                        const res = await fetch(`${url}?${params[0].key}=${v}`);
-                        return res.ok || "invalid";
+                        try {
+                          const { params, url } = f.validationAsync!;
+                          const res = await fetch(
+                            `${url}?${params[0].key}=${v}`
+                          );
+                          return res.ok || "invalid";
+                        } catch (err) {
+                          logger.error(err);
+                          return "validation failed";
```

Comment:
Nit: Maybe clarify that a little bit.
```suggestion
                          return "Validation of banking details failed unexpectedly";
```